scripts/sdk/base-manifest.json

```diff
@@ -1,15 +1,17 @@
 {
     "name": "hydrogen-view-sdk",
     "description": "Embeddable matrix client library, including view components",
-    "version": "0.0.9",
+    "version": "0.0.10",
     "main": "./lib-build/hydrogen.cjs.js",
     "exports": {
         ".": {
             "import": "./lib-build/hydrogen.es.js",
             "require": "./lib-build/hydrogen.cjs.js"
         },
         "./paths/vite": "./paths/vite.js",
-        "./style.css": "./asset-build/assets/index.css",
+        "./style.css": "./asset-build/assets/theme-element-light.css",
+        "./theme-element-light.css": "./asset-build/assets/theme-element-light.css",
```

Comment:
Could we avoid this if we just say in the SDK docs that the path to the css file is `hydrogen-view-sdk/assets/theme-element-light.css` rather than `hydrogen-view-sdk/theme-element-light.css`. Will be easier towards the future when adding more assets. Probably best to keep style.css for now for backwards compat though.

Reply:
> Could we avoid this if we just say in the SDK docs that the path to the css file is `hydrogen-view-sdk/assets/theme-element-light.css` rather than `hydrogen-view-sdk/theme-element-light.css`. Will be easier towards the future when adding more assets. 

Yes, we can! Should we do that in this PR or in a follow-up?

Reply:
The 👍 reaction is ambiguous on what option to do but I'm going to opt to fix this in a fast follow-up PR.

Reply:
wfm

Reply:
Addressed in https://github.com/vector-im/hydrogen-web/pull/746 ⏩